### PR TITLE
Dynamic Update for Upcoming Tasks GUI + JUnits for ParserTest, CommandManager, and HelpCommand

### PR DESCRIPTION
**Pull request by lmtaek**
Altered MainWindow slightly to handle updates to AssignedTask information within the Upcoming Tasks tab. Added/updated test classes to verify positive tests; will add negative tests if time allows.

### CODE REVIEW

File: src/main/java/duke/commands/CommandManager.java [hunk 1]
```diff
@@ -93,9 +93,8 @@ public static Command manageCommand(String userInput) throws DukeException {
             try {
                 return new DeleteAssignedTaskCommand(parser.parseUserInput()[0]);
             } catch (Exception e) {
-                throw new DukeException(CommandManager.class, "Please follow the `delete assigned task :#<patient id>"
-                        + " :#<task id>` or "
```
Comment (kkeejjuunn) [inline src/main/java/duke/commands/CommandManager.java]: This "CommandManager.class" is added for the logger. So do call this method instead.

File: src/main/java/duke/gui/MainWindow.java [hunk 2]
```diff
@@ -559,6 +559,12 @@ public void updateTableViews() {
         patientSearchBarListener();
         assignedTaskSearchBarListener();
         taskSearchBarListener();
+
+        try {
```
Comment (kkeejjuunn) [inline src/main/java/duke/gui/MainWindow.java]: Same for this DukeException.

Comment (lmtaek) [reply src/main/java/duke/gui/MainWindow.java]: I think you may have been referring to the other altered exception in the CommandManager class? This particular try/catch didn't exist until I added it in last night.

File: src/main/java/duke/commands/CommandManager.java (hunk 1 above)
Comment (lmtaek) [reply src/main/java/duke/commands/CommandManager.java]: I had to change the message itself due to its format being incorrect, but the CommandManager.class parameter has been added back in.

File: src/main/java/duke/gui/MainWindow.java (hunk 2 above)
Comment (kkeejjuunn) [reply src/main/java/duke/gui/MainWindow.java]: Good!

**Pull request merged**